src/app/header/header.component.html

```diff
@@ -6,7 +6,19 @@
   </div>
 
   <div class="col-md-4 col-sm-8" style="padding-top: 10px;">
+    <div>
+
+    </div>
     <div class="row font-weight-bold">
+      <!-- <span *ngFor="let link of menuarray">
+        <div class="col-md-12">
+          <span [routerLinkActive]="['active']">
+            <a routerLink="{{link.anchorLink}}">{{link.value}}</a>
+            <a [routerLink]="['/single-user']"> </a>
+            <a [routerLink]="['/multi-users']"></a>
+          </span>
+        </div>
+      </span> -->
```

Comment:
Line number 35 : why are you linking to unauthorized page?

Reply:
Currently we didn't have any page for assign users . Just we are loading that component, i removed now